OIPS/oip-draft-versioning.md

```diff
@@ -0,0 +1,41 @@
+---
+oip: 4
+title: Releases Versioning Specification
+author:  Francesco Pacella <f@ost.com>
+discussions-to: -
+status: Draft
+type: Meta
+category -
+created: 2018-11-23
+requires : -
+replaces : -
+---
+
+#Releases Versioning Specification
```

Comment:
After heading hashes, there should be a space (e.g., "#Releases" --> "# Releases"), otherwise it is rendered as simple text.